server/test/utils/setup-db.ts

```diff
@@ -0,0 +1,56 @@
+// check-db.ts
+
+import { PrismaClient } from '@prisma/client';
```

Comment:
setting up the db for the tests!

Reply:
@charlesBochet We've actually add creation of `metadata` schema in the `init.sql` file

Reply:
yes, but the init.sql is not called on test from the ci (we are using a postgres github action that cannot take an init.sql as parameter). Maybe we can move this to a ci-folder. It should not be needed locally indeed as we are using init.sql